jobs/config.json

```diff
@@ -10104,7 +10104,7 @@
       "--deployment=node",
       "--gcp-project=cri-c8d-pr-node-e2e",
       "--gcp-zone=us-central1-f",
-      "--node-args=--image-config-file=test/e2e_node/image-config.yaml --node-envs=PULL_REFS",
+      "--node-args=--image-config-file=test/e2e_node/image-config.yaml --image-config-dir=$GOPATH/src/github.com/kubernetes-incubator/cri-containerd --node-env=\"PULL_REFS=$(PULL_REFS)\"",
```

Comment:
hummm, I feel like you would have to replace `$GOPATH` with `go`, --image-config-dir does not know how to expand env vars

Reply:
The shell will expand it.

Reply:
there's no default variable expansion, I thought you were passing env name only, did not notice that changed to pass env value as well.

Reply:
You can try to move this to runner args which runs under bash

Reply:
Done.